pkg/cmd/codespace/create_test.go

```diff
@@ -0,0 +1,48 @@
+package codespace
+
+import (
+	"testing"
+
+	"github.com/cli/cli/v2/pkg/iostreams"
+)
+
+func TestBuildDisplayName(t *testing.T) {
+	tests := []struct {
+		name                 string
+		prebuildAvailability string
+		expectedDisplayName  string
+	}{
+		{
+			name:                 "prebuild availability is pool",
+			prebuildAvailability: "pool",
+			expectedDisplayName:  "4 cores, 8 GB RAM, 32 GB storage (Prebuild ready)",
+		},
+		{
+			name:                 "prebuild availability is blob",
+			prebuildAvailability: "blob",
+			expectedDisplayName:  "4 cores, 8 GB RAM, 32 GB storage (Prebuild ready)",
+		},
+		{
+			name:                 "prebuild availability is none",
+			prebuildAvailability: "none",
+			expectedDisplayName:  "4 cores, 8 GB RAM, 32 GB storage",
+		},
+		{
+			name:                 "prebuild availability is empty",
+			prebuildAvailability: "",
+			expectedDisplayName:  "4 cores, 8 GB RAM, 32 GB storage",
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			io, _, _, _ := iostreams.Test()
+			makeGrayText := io.ColorScheme().Gray
```

Comment:
We don't output color for iostreams in "Test" mode, so this Gray function will never do anything other than return the input argument unchanged. Therefore there is no real point in explicitly using it in tests